backup/setup/systems.json

```diff
@@ -32,7 +32,9 @@
 			"codeRedirectUri": "http://localhost:3030/api/v3/sso/oauth/0000d186816abba584714c92",
 			"authEndpoint": "https://iserv.n21.dbildungscloud.de/iserv/auth/auth",
 			"provider": "iserv",
-			"logoutEndpoint": "https://iserv.n21.dbildungscloud.de/iserv/auth/logout"
+			"logoutEndpoint": "https://iserv.n21.dbildungscloud.de/iserv/auth/logout",
```

Comment:
Please use "Schulserver" as alias further up

Reply:
done. Also changed it for dev instance.